demo/react-hooks-controlled-multiple.html

```diff
@@ -142,35 +135,42 @@ <h2>Controlled stack</h2>
     const [items2, setItems2] = useState([{ id: 'item-2-1', x: 0, y: 0, w: 1, h: 1 }, { id: 'item-2-2', x: 0, y: 1, w: 1, h: 1 }, { id: 'item-2-3', x: 1, y: 0, w: 1, h: 1 }])
 
     return (
-      <div style={{display: 'flex'}}>
-        <div style={{ display: 'flex', width: '50%' }}>
-          <ControlledStack
-            id='gs1'
-            items={items1}
-            addItem={(item) => {
-              setItems1(items => [...items, item])
-            }}
-            removeItem={(id) => {
-              setItems1(items => items.filter(i => i.id !== id))
-            }}
-          />
-        </div >
-        <div style={{ display: 'flex', width: '50%' }}>
-          <ControlledStack
-            id='gs2'
-            items={items2}
-            addItem={(item) => {
-              setItems2(items => [...items, item])
-            }}
-            removeItem={(id) => {
-              setItems2(items => items.filter(i => i.id !== id))
-            }}
-          />
+      <div>
+        <div style={{ display: 'flex', gap: '16px', marginBottom: '16px' }}>
+          <button onClick={() => setItems1(items => [...items, { id: `item-1-${items.length + 1}`, x: 2, y: 0, w: 2, h: 2 }])}>Add Item to 1 grid</button>
+          <button onClick={() => setItems2(items => [...items, { id: `item-2-${items.length + 1}`, x: 2, y: 0, w: 2, h: 2 }])}>Add Item to 2 grid</button>
         </div>
-      </div >
+        <div style={{ display: 'flex' }}>
+          <div style={{ display: 'flex', width: '50%' }}>
+            <ControlledStack
+              items={items1}
+              addItem={(item) => {
+                setItems1(items => [...items, item])
+              }}
+              removeItem={(id) => {
+                setItems1(items => items.filter(i => i.id !== id))
+              }}
+              changeItems={(items) => setItems1(items)}
```

Comment:
see comment above

Reply:
Hi, thank you for comments. I have fixed indentation